packages/spindle-ui/src/InlineNotification/InlineNotification.tsx

```diff
@@ -0,0 +1,174 @@
+import React, {
+  ReactNode,
+  FC,
+  Children,
+  cloneElement,
+  ReactElement,
+  HTMLAttributes,
+  useMemo,
+} from 'react';
+import { Button as SpindleButton } from '../Button/Button';
+import { TextButton as SpindleTextButton } from '../TextButton/TextButton';
+import { IconButton as SpindleIconButton } from '../IconButton/IconButton';
+
+type Variant = 'information' | 'confirm' | 'error';
+type Layout = 'inset' | 'full';
+
+type Props = {
+  children?: ReactElement;
+  variant?: Variant;
+  emphasis?: boolean;
+  layout?: Layout;
+  visible?: boolean;
+};
+
+const BLOCK_NAME = 'spui-InlineNotification';
+const DEFAULT_VARIANT = 'information';
+const DEFAULT_EMPHASIS = false;
+const DEFAULT_LAYOUT = 'inset';
+
+type ButtonVariant = 'neutral' | 'danger' | 'contained' | 'outlined';
+const computedButtonVariant = (
+  variant: Variant,
+  emphasis: boolean,
+): ButtonVariant => {
+  if (variant === 'confirm') {
+    return emphasis ? 'contained' : 'outlined';
+  } else if (variant === 'error') {
+    return 'danger';
+  } else {
+    return 'neutral';
+  }
+};
+
+const Frame: FC<Props> = ({
+  children,
+  variant = DEFAULT_VARIANT,
+  emphasis = DEFAULT_EMPHASIS,
+  layout = DEFAULT_LAYOUT,
+  visible = false,
+}) => {
+  return (
+    <div
+      className={[
+        `${BLOCK_NAME}`,
+        `${BLOCK_NAME}--${variant}`,
+        emphasis && `${BLOCK_NAME}--emphasized`,
+        layout === 'full' && `${BLOCK_NAME}--${layout}`,
```

Comment:
fullの時に`xxx--full`クラスを付与したいという話だと思うので、シンプルに以下でよさそうに思いました 🙆🏻‍♀️ 
```suggestion
        layout === 'full' && `${BLOCK_NAME}--full`,
```